app/controllers/stories.controller.js

```diff
@@ -24,6 +28,27 @@ exports.create = async (req, res) => {
       message: "An error occurred while creating the story.",
     });
   }
+};
 
+// Retrieve all stories from the database.
+exports.findAll = (req, res) => {
+  const storyId = req.query.storyId;
+  var condition = storyId
+  ? {
+      id: {
+          [Op.like]: `%${storyId}%`,
+      },
+  }
+  : null;
 
+  Story.findAll({where: condition, order: [["name", "ASC"]]})
+  .then((data) => {
+      res.send(data);
+    })
+    .catch((err) => {
+      res.status(500).send({
```

Comment:
before responding should the error be "logged" to the console?

Reply:
I have this addressed in my work that has been merged to main.